client/post-editor/editor-revisions/index.jsx

```diff
@@ -1,53 +1,45 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import PureRenderMixin from 'react-pure-render/mixin';
 import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import { partial } from 'lodash';
+import React, { PropTypes } from 'react';
 
-export default React.createClass( {
-
-	displayName: 'EditorRevisions',
-
-	mixins: [ PureRenderMixin ],
-
-	propTypes: {
-		adminUrl: React.PropTypes.string,
-		revisions: React.PropTypes.array
-	},
-
-	getDefaultProps() {
-		return {
-			revisions: []
-		};
-	},
-
-	render() {
-		if ( ! this.props.revisions || ! this.props.revisions.length ) {
-			return null;
-		}
-
-		const lastRevision = this.props.revisions[ 0 ];
-		const revisionsLink = this.props.adminUrl + 'revision.php?revision=' + lastRevision;
+/**
+ * Internal dependencies
+ */
+import { CHILD_SIDEBAR_REVISIONS } from '../editor-sidebar/util';
 
-		return (
-			<a className="editor-revisions"
-				href={ revisionsLink }
-				target="_blank"
-				rel="noopener noreferrer"
-				aria-label={ this.translate( 'Open list of revisions' ) }
-			>
-				<Gridicon icon="history" size={ 18 } />
-				{ this.translate(
-					'%(revisions)d revision',
-					'%(revisions)d revisions', {
-						count: this.props.revisions.length,
-						args: {
-							revisions: this.props.revisions.length
-						}
-					}
-				) }
-			</a>
-		);
+function EditorRevisions( { revisions = [], translate, toggleChildSidebar } ) {
+	if ( ! revisions || ! revisions.length ) {
+		return null;
 	}
-} );
+
+	return (
+		<button
```

Comment:
Removed the `a` because it's not a link anymore, went for a `button` to not load all the `Button` related CSS that would imply creating even more changes to the scss. I tried to minimise the amount of disruption there since it's a part that will be retouched when implementing @folletto status accordion redesign in a future PR.